### DISCUSSION TITLE
Questions about security

### DISCUSSION THREAD

**Discussion by ghost** [super-productivity/super-productivity#3798] [Q&A] [answered]
Does chromium automatically update to patch CVE's?
Because I have read that often Electron packages have old Chromium versions which impacts security.

And how much does the program follow Electron's security recommendations?
https://www.electronjs.org/docs/latest/tutorial/security

I found these links that could be interesting
https://github.com/reZach/secure-electron-template
https://github.com/doyensec/electronegativity

**Comment by ghost** [comment]
@johannesjo, sorry to bother you I was wondering if you could look into this as I am worried the post might get lost otherwise.

**Comment by johannesjo** [comment, marked as answer]
Sorry @tegridyfarmer ! An answer would take a lot of time and to be honest I rather spent it working on something else. The code is open source, so everyone can check for themselves how the actual implementation would like.

I can offer a very broad answer which is that I am a web developer with a frontend focus and no security expert. I tried to follow the best practices as much as possible and I update electron often, but I can offer no guarantees.

Personally I feel quite safe using the app.

**Reply by ghost** [reply]
Understandable!

**Reply by ghost** [reply]
Should I make an issue to check for the mentioned good practices in https://www.electronjs.org/docs/latest/tutorial/security @johannesjo? So maybe at a later date you or someone else can look into it?

**Reply by johannesjo** [reply]
Yes, that is a good idea! Thank you! :)
